tests/unit/test_agent_skill.py

```diff
@@ -781,7 +781,7 @@ def test_file_editor_create(tmp_path):
     assert result is not None
```

Comment:
I'm thinking if we can start moving these unit tests out of OpenHands repo and into openhands-aci 🤔

Reply:
Done!